base/system/winlogon/winlogon.c

```diff
@@ -21,6 +21,29 @@ PWLSESSION WLSession = NULL;
 
 /* FUNCTIONS *****************************************************************/
 
+/**
+ * @brief
+ * Duplicates the given string, allocating a buffer on the heap.
+ **/
+PWSTR
+WlStrDup(
+    _In_opt_ PCWSTR String)
+{
+    PWSTR ptr;
+
+    if (!String)
+        return NULL;
+
+    ptr = RtlAllocateHeap(RtlGetProcessHeap(), 0,
+                          (wcslen(String) + 1) * sizeof(WCHAR));
+    if (!ptr)
+        return NULL;
+    /***/RtlFillMemory(ptr, (wcslen(String) + 1) * sizeof(WCHAR), 0xAB);/***/
+    wcscpy(ptr, String);
```

Comment:
```suggestion
        return NULL;

    wcscpy(ptr, String);
```

Reply:
Heh thanks, I forgot to remove this debugging code